### DISCUSSION TITLE
Erstellung Übersicht MQTT Werte  für Nicht-Senic Systeme

### DISCUSSION THREAD

**Discussion by WolfImBusch** [solectrus/solectrus#3283] [Ideas]
Ich konnte Solectrus mit Hilfe von Georg mit einem Fronius Wechselrichter, BYD Batterie und Keba P30C Wallbox erfolgreich in Betrieb nehmen. Das Zusammensuchen der über MQTT verteilten Werte (bei mir über HA und evcc) für diese Komponenten hat mich ein bisschen Zeit und Nerven gekostet. Wie wäre es eine Übersicht auf der Solectrus Homepage aufzunehmen, um dies für andere Benutzer zu erleichtern? Viele Grüße Alex

**Comment by ledermann** [comment]
An Möglichkeiten zur Vereinfachung der Installation von SOLECTRUS bin ich immer sehr interessiert. Gerade, wenn MQTT im Spiel ist, kann es knifflig werden. Jedes Gerät, das Daten liefert, hat da seine Eigenheiten.

Wie könnte denn eine solche Hilfestellung aussehen? Welcher Messwert in welchem MQTT-Topic liegt, kann ja leider bei jedem User unterschiedlich sein, auch wenn man das gleiche Gerät hat. Manche Hersteller bringen z.B. die Seriennummer mit in den Topic-Namen.

Was ich mir vorstellen könnte, wäre eine Sammlung von Texten beispielsweise im [Wiki](https://github.com/solectrus/solectrus/wiki/), in der einzelne User Ihre Erfahrung mit der Anbindung eines bestimmten Geräts beschreiben. Also z.B. jeweils ein kleiner Erfahrungsbericht der Art "Anbindung Fronix HighPower v4.5 an SOLECTRUS". 

Oder woran hast du gedacht?

**Comment by norbertle** [comment]
Ich glaube das es wirklich hilft wenn man Beispiel inkl. Anleitung hat wie man andere Systeme noch mit einbindet. Spontan fällt mir da z. B. die Werte ein welche ich über einen optischen Lesekopf vom Zähler auslesen kann. Also 1.8.0 und 2.8.0.

**Reply by ledermann** [reply]
Interessant, du liest tatsächlich Netzbezug und Einspeisung vom Stromzähler über einen optischen Lesekopf aus und bringst die Daten zu SOLECTRUS? Ja, für solche Dinge wären spezialisierte Anleitungen sicher nützlich.

**Reply by norbertle** [reply]
Nein das mache ich nicht.

Aber das wäre ein Wunsch das zu machen. Der Lesekopf kann MQTT. Dann müsste man nur einen MQTT Brocker im Docker installieren und könnte das ja dann in die DB schreiben. So auf jeden Fall mein Verständnis.
Wenn ich das gehabt hätte dann könnte ich mit den Zählerständen des Stromzählers jetzt nach dem 24 Stunden Telefon/Internet Ausfall das in Solectrus fehlende Delta berechnen. Weil der Speicher läuft ja weiter wenn mal kurzfristig das Internet weg ist.

**Reply by frizzle0815** [reply]
Ich lese meinen Stromzähler über einen Lesekopf von Lingg&Janke aus, der über einen Weinzierl KNX IP BAOS die Daten an meinen Gira Homeserver schickt. Von dort wird an den Mosquitto MQTT Server gesendet, der die Daten meiner Wallbox und Solectrus Verfügung stellt.

Das sind mir an sich etwas zu viele Komponenten, die ausfallen könnten, aber es funktioniert bislang problemlos :-)

**Comment by ledermann** [comment]
Ich habe mal das [Wiki](https://github.com/solectrus/solectrus/wiki/) freigeschaltet, sodass jeder dort etwas schreiben kann. 

Wohin das führt, wird sich zeigen. Möglicherweise müssen die Texte (sofern überhaupt welche entstehen) später nachbearbeitet oder aufbereitet werden, oder manches ist dann vielleicht besser im Konfigurator aufgehoben. Mal abwarten :-)
